src/main/java/io/deephaven/benchmark/api/BenchTable.java

```diff
@@ -41,6 +41,20 @@ public BenchTable add(String name, String type, String valuesDef) {
         return this;
     }
 
+    /**
+     * Add a column definition for the table schema
+     * 
+     * @param name the name of the column
+     * @param type the type of the column ( <code>string | long | int | double | float</code> )
+     * @param valuesDef range or combination of range and string
+     * @param distribution the name of the distribution ( <code>linearConv</code> )
```

Comment:
Would it be better to use an enum for this?

Reply:
I've struggled back and forth with this one.  I chose to use strings because I worried that "locking it down" would make it harder to make it more modular later.  For example, we could eventually have user-supplied distributions or generator types that plug-in.  We could also eventually allow configuration of generators from resource files, or usage of the Bench API from python, and using strings for those items could minimize the needs for wrappers.  

So it seems like using strings for now gives some flexibility for what we don't know we need yet.  Is that a bad excuse?